Source/Pawnmorphs/Esoteria/Hediffs/Comp_MutationSeverityAdjust.cs

```diff
@@ -208,10 +204,10 @@ public void OnStageChanged(HediffStage oldStage, HediffStage newStage)
         /// </summary>
         public void RecalcAdjustSpeed()
         {
-            StatAdjust.Recalculate();
-            MutationAdaptability.Recalculate();
-            IsReverting.Recalculate();
-            ShouldRemove.Recalculate();
+            //StatAdjust.Recalculate();
+            //MutationAdaptability.Recalculate();
+            //IsReverting.Recalculate();
+            //ShouldRemove.Recalculate();
         }
```

Comment:
This method should either be deprecated or changed to manually flush the cache (depending on what uses it and how important it is for the update to be immediate)